app/modules/inventory/components/shelf_dot.svelte

```diff
@@ -0,0 +1,57 @@
+<script>
+  import { isOpenedOutside } from '#lib/utils'
+  import { loadShelfLink } from '#shelves/components/lib/shelves'
+  import { getColorHexCodeFromModelId } from '#lib/images'
+  import screen_ from '#lib/screen'
+
+  export let shelf
+
+  const { name, _id } = shelf
+
+  const colorHexCode = shelf.color || `#${getColorHexCodeFromModelId(_id)}`
+
+  function onShelfDotClick (e, shelf) {
+    if (isOpenedOutside(e)) return
+    loadShelfLink(shelf)
+    e.preventDefault()
+  }
```

Comment:
Now that the shelf dot is a component, the shelf is in `onShelfDotClick` scope
```suggestion
  function onShelfDotClick (e) {
    if (isOpenedOutside(e)) return
    loadShelfLink(shelf)
    e.preventDefault()
  }
```